scss/civicrm/_variables.scss

```diff
@@ -27,6 +29,11 @@ $crm-dark-gray-color: #70716b;
 $crm-green-color: #B0D730;
 $crm-light-gray-color: #818c97;
 
+$crm-columnheader-title-color: #4D5663;
```

Comment:
Do not introduce any new color in system then we already have, use colors from pre-defined palette